backend/tests/test_get.py

```diff
@@ -0,0 +1,100 @@
+import pytest
```

Comment:
Maybe rename this file to test_get_by_keyword? That way, future tests for future GET routes can be put into other files without confusion. I named my test files by their corresponding function names in app.py, but you can shorten the test file name to test_search or something (maybe even rename the app.py function if you want).

Reply:
Completed